docs/how-to-guides/data-basics/3-building-an-edit-form.md

```diff
@@ -0,0 +1,521 @@
+# Building an edit form
+
+This part is about adding an *Edit* feature to our app. Here's a glimpse of what we're going to build:
+
+![](./media/edit-form/form-finished.png)
+
+### Step 1: Add an _Edit_ button
+
+We can't have an *Edit* form without an *Edit* Button, so let's start by adding one to our `PagesList` component:
```

Comment:
```suggestion
We can't have an *Edit* form without an *Edit* button, so let's start by adding one to our `PagesList` component:
```

Nit: lowercase unless we're refering to the component in which case I'd recommend `<Button>`.